ephios/core/views/settings.py

```diff
@@ -1,28 +1,36 @@
 import typing
 
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib.auth.views import PasswordChangeView
 from django.contrib.messages.views import SuccessMessageMixin
-from django.urls import reverse
+from django.urls import reverse, reverse_lazy
 from django.utils.translation import gettext as _
 from django.views.generic import FormView
 from dynamic_preferences.forms import global_preference_form_builder
 
+from ephios.core.forms.users import UserNotificationPreferenceForm
 from ephios.core.signals import administration_settings_section
 from ephios.extra.mixins import StaffRequiredMixin
 
 
 def get_available_administration_settings_sections(request):
-    sections = [
-        {
-            "label": _("General"),
-            "url": reverse("core:settings_general"),
-            "active": request.resolver_match.url_name == "settings_general",
-        },
-        {
-            "label": _("Event types"),
-            "url": reverse("core:settings_eventtype_list"),
-            "active": request.resolver_match.url_name.startswith("settings_eventtype"),
-        },
-    ]
+    sections = []
+    if request.user.is_staff:
+        sections.append(
+            {
+                "label": _("ephios instance"),
+                "url": reverse("core:settings_instance"),
+                "active": request.resolver_match.url_name == "settings_instance",
+            }
+        )
+    if request.user.has_perm("core.view_eventtype"):
```

Comment:
I mean it probably doesn't matter, but people will get 403s if they click on one of the buttons and they don't have permission to change.

Reply:
Well the view that is behind the URL requires the view permission, so I thought it would be a good idea to check for that permission. Maybe there will be some setup where it could make sense to only allow people to view those types? For the 403 argument it would be better to hide the buttons in the list view if the user doesn't have the appropriate permissions

Reply:
I think we can leave it for now and hide the buttons if the situation ever comes up.